common/config/src/raw.rs

```diff
@@ -445,6 +447,7 @@ impl Default for LogConfig {
             k8s_metadata_include: None,
             k8s_metadata_exclude: None,
             log_metric_server_stats: None,
+            clear_cache_interval: Some(3600 * 3), // 3 hours
```

Comment:
Was 3 hours chosen at random or is there a significant build up over that period?

Reply:
build up is unconstrained. depends on file event rate. by consensus.